test/unit/DrbbContainer.test.js

```diff
@@ -0,0 +1,113 @@
+/* eslint-env mocha */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import { spy, stub } from 'sinon';
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+
+import DrbbContainer from './../../src/app/components/Drbb/DrbbContainer';
+import { mockRouterContext } from '../helpers/routing';
+
+describe('DrbbContainer', () => {
+  let component;
+  const context = mockRouterContext();
+
+  describe('initial render', () => {
+    before(() => {
+      component = shallow(<DrbbContainer />, {
+        context,
+        disableLifecycleMethods: true,
+      });
+    });
+
+    it('should render the loading component', () => {
+      expect(component.find('.drbb-loading-layer')).to.be.defined;
+      expect(component.find('.drbb-container')).to.have.length(1);
+    });
+
+    it('should be wrapped in a .drbb-container class', () => {
```

Comment:
In general I don't like checking for classes, because they can be brittle.  If the thing you want to check for is "This class makes stuff green and it's important stuff is green", then that should go in visual regression testing (which we don't do automatically at NYPL) but checking for a class doesn't confirm that anyway.  

If it's checking for some wrapper component though, you should check for the existence of that component, rather than the existence of the class.